app/auth-portal/src/pages/WorkspaceAuthTokensPage.vue

```diff
@@ -175,8 +199,14 @@ const createAuthToken = useAsyncState(
 /** Name of token to create */
 const createAuthTokenName = ref("");
 
+/** Token modal */
+const tokenDisplayModalRef = ref<InstanceType<typeof Modal> | null>(null);
+
+const tokenCopied = ref(false);
 async function copyToken() {
-  if (createAuthToken.state.value)
-    await navigator.clipboard.writeText(createAuthToken.state.value);
+  if (createAuthToken.state.value) {
+    await navigator.clipboard?.writeText(createAuthToken.state.value);
+  }
+  tokenCopied.value = true;
```

Comment:
It seems unlikely to trigger as a bug, but this should probably be inside that `if`